src/Sulu/Bundle/ContactBundle/Api/Account.php

```diff
@@ -1166,4 +1086,24 @@ public function getMedias()
 
         return $medias;
     }
+
+    /**
+     * Get categories
+     *
+     * @return Category[]
+     * @VirtualProperty
+     * @SerializedName("categories")
+     * @Groups({"fullAccount"})
+     */
+    public function getCategories()
```

Comment:
no not needed ... set categories is done by the manager which uses the entitites